apisix/plugins/basic-auth.lua

```diff
@@ -30,7 +30,12 @@ local consumers_lrucache = core.lrucache.new({
 local schema = {
     type = "object",
     title = "work with route or service object",
-    properties = {},
+    properties = {
+        hide_auth_header = {
+            type = "boolean",
+            default = true,
```

Comment:
I think the default value should be false.

Reply:
Done.